css/style.css

```diff
@@ -33,11 +144,113 @@ body {
   text-transform: uppercase;
 }
 
-.product-item__name {
+.market-section__subtitle {
+  text-align: center;
+  margin: 51px 0 16px 0;
+}
+
+.products {
+  display: flex;
+  justify-content: space-between;
+}
+
+.products__product-card {
+  margin: 0 0 25px 0;
+  /* background-color: rgb(160, 32, 32); */ /* для проверки */
+}
+
+.product-card {
+  width: 230px;
+  position: relative;
+  padding: 0 12px;
+  /* height: 261px;   */
+  /* border: 1px solid rgb(23, 125, 209); */
+}
+
+.product-card::after {
+  content: "";
+  position: absolute;
+  top: 40px;
+  right: 108%; /* Найти лучшее решение */
+  /* display: inline-block; */
+  width: 1px;
+  height: 100px;
+  background:  #000000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654147)